.github/workflows/benchmark_template.yml

```diff
@@ -8,29 +8,62 @@ on:
       test-flags:
         required: true
         type: string
+      external-data-json-path:
+        type: string
+
+permissions:
+  # deployments permission to deploy GitHub pages website
+  deployments: write
+  # contents permission to update benchmark contents in gh-pages branch
+  contents: write
 
 env:
   CGO_ENABLED: 0
 
 jobs:
   benchmarks:
     if: ${{ github.repository == 'gnolang/gno' }}
-    runs-on: [self-hosted, Linux, X64, benchmark-v1]
+    runs-on: ubuntu-latest
```

Comment:
I think we should keep a custom runner here.

Reply:
we just removed this resource, but we are planning on getting them back differently.
Unfortunately the target machine has been sleeping for months and it was quite huge in terms of HW.

Reply:
I posted the doubt, because basically I agree with you on meaningfulness of same run on GH runners